python/jupyter_suggestions_core/schema/registry.json

```
@@ -0,0 +1,17 @@
+{
+  "jupyter.lab.setting-icon": "jupyter-suggestions:hintIcon",
+  "jupyter.lab.setting-icon-label": "Jupyter Suggestions",
+  "jupyter.lab.transform": true,
+  "title": "Jupyter Suggestions",
+  "description": "Jupyter Suggestions settings.",
```

Maybe "Suggestions" is enough when displayed in the UI?

```suggestion
  "jupyter.lab.setting-icon-label": "Suggestions",
  "jupyter.lab.transform": true,
  "title": "Suggestions",
  "description": "Suggestions settings.",
```